src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (Ball ball : balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
+        }
```

Comment:
The loop is not correctly populating the 'balls' array. The enhanced for loop creates a temporary variable 'ball' which is not affecting the array. Use a standard for loop to assign new Ball objects to the 'balls' array elements.